packages/grafana-data/src/types/legacyEvents.ts

```diff
@@ -0,0 +1,44 @@
+import { DataQueryError, DataQueryResponseData } from './datasource';
+import { AngularPanelMenuItem } from './panel';
+import { DataFrame } from './dataFrame';
+import { eventFactory } from '../events/eventFactory';
+import { BusEventBase, BusEventWithPayload } from '../events/types';
+
+export type AlertPayload = [string, string?];
+export type AlertErrorPayload = [string, (string | Error)?];
+
+export const AppEvents = {
+  alertSuccess: eventFactory<AlertPayload>('alert-success'),
+  alertWarning: eventFactory<AlertPayload>('alert-warning'),
+  alertError: eventFactory<AlertErrorPayload>('alert-error'),
+};
+
+export const PanelEvents = {
```

Comment:
Do we need to put this in `@grafana/data`?  Can the legacy things be internal packages?

Reply:
Some are used angular panel plugins

Reply:
and this is already in grafana/data, so would break plugins if we removed it